bcda-site-static/user_guide.md

```diff
@@ -624,3 +625,163 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
 	}
 }
 ```
+## Beneficiary Coverage Data
+The process of retrieving coverage data is very similar to all of the other exporting operations supported by this api.
```

Comment:
Looking throughout the user guide, we capitalize "api".  Requesting that we change "api" to "API" here.

Reply:
fixed